### DISCUSSION TITLE
How do you specify nexus credentials ?

### DISCUSSION THREAD

**Discussion by laurentperez** [jbangdev/jbang#1208] [General]
Hello

Asking here because I couldn't find information in the docs

I noticed https://github.com/jbangdev/jbang/commit/e09464b1d3445c58fc3d007b976a0a7076a11294

How would you enable a custom repository for // DEPS ? Or would it read .gradle/gradle.properties or .m2/settings.xml ?

**Comment by maxandersen** [comment]
Should work if specified in m2 settings. If not that's a bug.
